prov/verbs/src/verbs_ep.c

```diff
@@ -77,7 +77,7 @@ ssize_t vrb_post_recv(struct vrb_ep *ep, struct ibv_recv_wr *wr)
 	if (!ctx)
 		goto unlock;
 
-	OFI_DBG_SET(ctx->ep, ep);
+	ctx->ep = ep;
```

Comment:
The lack of this change was the reason for the previous crashes in the release build but not debug.